src/global.css

```diff
@@ -29,4 +29,14 @@
 
 @keyframes l24{
   100%{background-position: left}
-}
\ No newline at end of file
+}
+
+.input-style {
```

Comment:
Used tailwind classes !! For all the inputs in the entire project same classes are used so instead of writing these classes again & again i just gave the whole class a name & that is better to give the input only classname input-style (Which increase code-redability ,code-reusability and better maintainance).